### PR TITLE
[site] image container size to 50%

### LINKED ISSUE #206: [Site] Unequal split of image and adjoining description on main page
**Description**

The meshery nighthawk performance chart image is bigger than 50% of its container.

**Expected Behavior**

It should be adjusted to be 50% of its container.

**Screenshots**
Screen Shot 2021-11-18 at 1 42 41 PM

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by akshitarora921**
Signed-off-by: Akshit <akshitarora9211@gmail.com>

**Description**
The meshery nighthawk performance chart image is not 50% of its container.

This PR fixes #206 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍 great work @akshitarora921

**Review by YashKamboj — COMMENTED** [review]
@akshitarora921  I think the photo should come below the description when the screen size decreases , same for the image below , this issue is already there , we can solve it here only . @Abhijay007 views???

**Review by Neilblaze — CHANGES_REQUESTED** [review]
The changes aren't populating the exact update which needs to be solved #206. Follow [this](https://github.com/layer5io/getnighthawk/issues/206#issuecomment-1013859380).

**Comment by Abhijay007** [comment]
> @akshitarora921 I think the photo should come below the description when the screen size decreases , same for the image below , this issue is already there , we can solve it here only . @Abhijay007 views???

ya, this can be solved just need to set  `flex-direction: column-reserve`  on styling properties of class `.custom-desc-div `

**Comment by akshitarora921** [comment]
> The changes aren't populating the exact update which needs to be solved #206. Follow [this](https://github.com/layer5io/getnighthawk/issues/206#issuecomment-1013859380).

This is not in the issue description thanks for pointing it out. I have fixed it by wrapping the data around the `container` class.

**Comment by Abhijay007** [comment]
> > The changes aren't populating the exact update which needs to be solved #206. Follow [this](https://github.com/layer5io/getnighthawk/issues/206#issuecomment-1013859380).
> 
> This is not in the issue description thanks for pointing it out. I have fixed it by wrapping the data around the `container` class.

@akshitarora921 don't forget to signoff DCO in your next commit :) it is good practice

**Comment by akshitarora921** [comment]
In the VS Code, I have changed the settings but I have made the last commit using `github.dev` and I forgot to add DCO.

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

File: docs/index.html [hunk 1]
```diff
@@ -93,8 +93,7 @@ <h1 class="desc-h1">Nighthawk and Meshery</h1>
       </div>
     <div class="nighthawk-meshery-div">
       <div class="container description-div description-div-grpc">
-        <div class="custom-desc-div">
-    
+        <div class="">
```
Comment (YashKamboj) [inline docs/index.html:96]: @akshitarora921 why did you removed the class??

Comment (akshitarora921) [reply docs/index.html:96]: It was not doing anything as it only have one child tag, so `display:flex` or `flex-direction: column-reverse` were not making any impact.

**Review by NikhilSharma03 — APPROVED** [review]
LGTM :+1:

File: docs/index.html (hunk 1 above)
Comment (adithyaakrishna) [inline docs/index.html:96]:
```suggestion
        <div>
```

Comment (adithyaakrishna) [reply docs/index.html:96]: @akshitarora921 Could you please make the small change as suggested above? Thanks for your contribution 🎉🚀

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 1 above)
Comment (akshitarora921) [reply docs/index.html]: @adithyaakrishna  Done

**Comment by leecalcote** [comment]
Yay, @akshitarora921! 🎈

**Pull request merged**